Design database

YutakaIzeki opened the pull request:
# WHAT
Chat spaceのデータベースを設計

# WHY
chat spaceを作るため

YutakaIzeki committed:
Edit README.md

YutakaIzeki committed:
Add_space_to_README

miyagit reviewed (commented):
中間テーブルのアソシエーションが抜けているので修正の方法がわからなければ、メンターに質問しましょう。

YutakaIzeki commented:
> 中間テーブルのアソシエーションが抜けているので修正の方法がわからなければ、メンターに質問しましょう。

YutakaIzeki committed:
Edit README.md

Edit Association in members table

TOMARUMARU reviewed (commented):
中間テーブルを用いた多対多のアソシエーションについてしっかり調べて修正をお願いします

YutakaIzeki committed:
Edit README.md

Add Association to users table and group table

YutakaIzeki commented:
よろしくお願いいたします

TOMARUMARU commented on chat-space/README.md:
```diff
@@ -22,3 +22,62 @@ Things you may want to cover:
+
+|Column|Type|Option|
+|------|----|------|
+|name|string|null: false, add_index :users, :name|
```
```add_index :users, :name```ではなく```index: true```で大丈夫です
uniqueもつけましょう

YutakaIzeki replied:
修正しました

TOMARUMARU commented on chat-space/README.md:
```diff
@@ -22,3 +22,62 @@ Things you may want to cover:
+
+|Column|Type|Option|
+|------|----|------|
+|body|text|null: true|
```
contentなどの名前が良さそうです

YutakaIzeki replied:
修正しました

TOMARUMARU commented on chat-space/README.md:
```diff
@@ -22,3 +22,62 @@ Things you may want to cover:
+|Column|Type|Option|
+|------|----|------|
+|body|text|null: true|
+|image|string|null: true|
```
画像、テキストどちらかで遅れる仕様なのでcontent、imageどちらもnull: falseは外しましょう

YutakaIzeki replied:
修正しました

YutakaIzeki committed:
Edit README.md

Edit users table
  1. add_index :users, :name →index: true

  1. Add to users table
      unique: true

Edit message table
 1. Body → content

 1. Delete null: true

TOMARUMARU reviewed (approved):
LGTMです。お疲れ様でした
